src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -238,10 +230,8 @@ public AzureAppConfigurationOptions UseFeatureFlags(Action<FeatureFlagOptions> c
                 var featureFlagFilter = featureFlagSelector.KeyFilter;
                 var labelFilter = featureFlagSelector.LabelFilter;
 
-                if (!_kvSelectors.Any(selector => selector.KeyFilter == featureFlagFilter && selector.LabelFilter == labelFilter))
-                {
-                    Select(featureFlagFilter, labelFilter);
-                }
+                // For FeatureFlags, FeatureFlagOptions.Select should have higher precedence than AzureAppConfigurationOptions.Select
```

Comment:
Do we encourage users to use `AzureAppConfigurationOptions.Select` with FeatureFlagMarker to select feature flags?
```c#
options.useFeatureFlags(ffOptions => {
  ffOptions.Select("Feature", "labelA");
});
...
options.Select(".appconfig.featureflag/Feature", "labelB");
```

Just to confirm that in this case, labelB wins, right?

Reply:
> Do we encourage users to use AzureAppConfigurationOptions.Select with FeatureFlagMarker to select feature flags?

We dont encourage it, but we dont block them from doing it either.

> Just to confirm that in this case, labelB wins, right?

Yes that is correct. We load key-values and feature flags based on the insertion order in `AzureAppConfigurationOptions.KeyValueSelectors`.

Reply:
Then I would simply remove this line of comments. Next line works because the correct dedup logic is already in `AzureAppConfigurationOptions.Select`.